src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua

```diff
@@ -0,0 +1,833 @@
+function Adjustable.Container:onMove (label, event)
+    if self.locked then
+        if label.cursorShape ~= 0 then 
+            label:resetCursor()
+        end
+        return
+    end
+    if adjustInfo.move == nil then
+        adjust_Info(self, label, event)
+    end
+    if adjustInfo.x and adjustInfo.name == label.name then
+        self:adjustBorder()
+        local x, y = getMousePosition()
+        local winw, winh = getMainWindowSize()
+        local x1, y1, w, h = self.get_x(), self.get_y(), self:get_width(), self:get_height()
+        if (self.container) and (self.container ~= Geyser) then
+            x1,y1 = x1-self.container.get_x(), y1-self.container.get_y()
+            winw, winh = self.container.get_width(), self.container.get_height()
+        end
+        local dx, dy = adjustInfo.x - x, adjustInfo.y - y
+        local max, min = math.max, math.min
+        if adjustInfo.move then
+            label:setCursor("ClosedHand")
+            local tx, ty = max(0,x1-dx), max(0,y1-dy)
+            tx, ty = min(tx, winw - w), min(ty, winh - h)
+            tx = make_percent(tx/winw)
+            ty = make_percent(ty/winh)
+            self:move(tx, ty)
+            --[[
+            -- automated lock on border deactivated for now
+            if x1-dx <-5 then self:attachToBorder("left") end
+            if y1-dy <-5 then self:attachToBorder("top") end
+            if winw - w < tx+0.1 then self:attachToBorder("right") end
+            if winh - h < ty+0.1 then self:attachToBorder("bottom") end--]]
+        elseif adjustInfo.move == false then
+            local w2, h2, x2, y2 = w - dx, h - dy, x1 - dx, y1 - dy
+            local tx, ty, tw, th = x1, y1, w, h
+            if adjustInfo.top then
+                ty, th = y2, h + dy
+            elseif adjustInfo.bottom then
+                th = h2
+            end
+            if adjustInfo.left then
+                tx, tw = x2, w + dx
+            elseif adjustInfo.right then
+                tw = w2
+            end
+            tx, ty, tw, th = max(0,tx), max(0,ty), max(10,tw), max(10,th)
+            tw, th = min(tw, winw), min(th, winh)
+            tx, ty = min(tx, winw-tw), min(ty, winh-th)
+            tx = make_percent(tx/winw)
+            ty = make_percent(ty/winh)
+            self:move(tx, ty)
+            local minw, minh = 0,0
+            if self.container == Geyser and not self.noLimit then minw, minh = 75,25 end
+            tw,th = max(minw,tw), max(minh,th)
+            tw,th = make_percent(tw/winw), make_percent(th/winh)
+            self:resize(tw, th)
+            shrink_title(self)
+        end
+        adjustInfo.x, adjustInfo.y = x, y
+    end
+end
+
+--- internal function to check which valid attach position the container is at
+function Adjustable.Container:fPos()
+    local winw, winh = getMainWindowSize()
+    local fpos = {}
+    if  (winh*0.8)-self.get_height()<= self.get_y()  then  table.insert(fpos,"bottom") end
```

Comment:
```suggestion
    if  (winh*0.8)-self.get_height()<= self.get_y()  then fpos[#fpos+1] = "bottom" end
```

This is quicker, change others too